sofl/game_factory.py

```
@@ -0,0 +1,50 @@
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from typing import Any
+
+from sofl.game import Game
+from sofl.game_data import GameData
+from sofl.onlinefix_game import OnlineFixGameData
+
+class GameFactory:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)